File: app/src/main/java/cc/smartcash/wallet/Models/SmartApiItem.java
package cc.smartcash.wallet.Models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import java.io.Serializable;

@JsonIgnoreProperties(ignoreUnknown = true)
public class SmartApiItem implements Serializable {

    private String updated;

    private SmartApiCurrency currencies;

    private String ticker;

    private String object;

    private String created;

    private String exchange;

    private String id;

    private String name;

    public String getUpdated() {
        return this.updated;
    }

    public void setUpdated(String updated) {
        this.updated = updated;
    }

    public SmartApiCurrency getCurrencies() {
        return this.currencies;
    }

    public void setCurrencies(SmartApiCurrency currencies) {
        this.currencies = currencies;
    }

    public String getTicker() {
        return this.ticker;
    }

    public void setTicker(String ticker) {
        this.ticker = ticker;
    }

    public String getObject() {
        return this.object;
    }

    public void setObject(String object) {
        this.object = object;
    }

    public String getCreated() {
        return this.created;
    }

    public void setCreated(String created) {
        this.created = created;
    }

    public String getExchange() {
        return this.exchange;
    }

    public void setExchange(String exchange) {
        this.exchange = exchange;
    }

    public String getId() {
        return this.id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return this.name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/cc/smartcash/smarthub/Models/SmartApiItem.kt
package cc.smartcash.smarthub.Models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

import java.io.Serializable

@JsonIgnoreProperties(ignoreUnknown = true)
data class SmartApiItem(

        var updated: String? = null,

        var currencies: SmartApiCurrency? = null,

        var ticker: String? = null,

        var `object`: String? = null,

        var created: String? = null,

        var exchange: String? = null,

        var id: String? = null,

        var name: String? = null

) : Serializable
